core/src/network/handler/lists.rs

```diff
@@ -90,3 +122,67 @@ pub fn handle_get_gossip_list(get_list_data: GetListData, context: Arc<Context>)
         })
         .expect("Could not spawn thread for creating of gossip list");
 }
+
+#[cfg(test)]
+pub mod tests {
+    use super::*;
+    use crate::workflows::author_entry::author_entry;
+    use crate::nucleus::actions::tests::*;
+    use holochain_core_types::{
+        entry::{Entry, test_entry_with_value},
+    };
+    use holochain_persistence_api::cas::content::{AddressableContent, Address};
+    use std::{thread, time};
+    
+    #[test]
+    fn test_can_get_chain_header_list() {
```

Comment:
@lucksus These two tests show the loading of the header entries and their aspects working fine. I am wondering if the issue is happening only at startup when the Dna and Agent headers are in the chain but the header entries are not committed yet.